src/utils/form.tsx

```diff
@@ -225,6 +225,15 @@ export function afterConnectDefaultValueSet<T>(component: FormComponent<T>, valu
   component.defaultValue = value;
 }
 
+const hiddenInputChangeHandler = (event: Event) => {
+  event.target.dispatchEvent(
+    new CustomEvent("calciteInternalHiddenInputChange", {
+      bubbles: true,
+      composed: true
```

Comment:
It _think_ we don't need the event to be `composed` since it's emitted and listened to on the light DOM.